test/metabase/lib/drill_thru/zoom_test.cljc

```diff
@@ -76,6 +78,21 @@
                   :object-id (get-in lib.drill-thru.tu/test-queries ["ORDERS" :unaggregated :row "ID"])
                   :many-pks? false}}))
 
+(deftest ^:parallel returns-zoom-source-card-test
+  (let [metadata-provider (lib.tu/metadata-provider-with-cards-for-queries
+                           meta/metadata-provider
+                           [(lib/query meta/metadata-provider (meta/table-metadata :orders))])
+        query (lib/query metadata-provider (lib.metadata/card metadata-provider 1))]
+    (lib.drill-thru.tu/test-returns-drill
```

Comment:
We should also test that the query returned by applying this drill is what we expect. See the `lib.drill-thru.tu/test-drill-application` helper instead of `test-returns-drill`. There aren't any uses in this file but there are plenty of examples.